Packs/Slack/README.md

```diff
@@ -744,7 +744,16 @@ new incident [details]
 mirror incident-id
 </pre>
 <h3>Notifications</h3>
-<p>The integration sends notifications according to the configured notification settings found in the user preferences page in Cortex XSOAR, and additionally to the pre determined channel dedicated for incident notifications (according to the integration configuration).</p>
+<p>The integration sends notifications according to the configured notification settings found in the user preferences page in Cortex XSOAR, and additionally to the pre determined channel dedicated for incident notifications (according to the integration configuration).
```

Comment:
why do we have the readme here as well ?

Reply:
It's Slack + SlackAsk

Reply:
ok so lets keep this readme empty and have one integration for one for the script